### DISCUSSION TITLE
How to Pinch and Zoom in Native iOS Application

### DISCUSSION THREAD

**Discussion by dvngntfs** [webdriverio/webdriverio#9255] [Q&A]
The closest example I see is

   ```
browser.actions([{
        "type": "pointer",
        "id": "finger1",
        "parameters": {"pointerType": "touch"},
        "actions": [
            {"type": "pointerMove", "duration": 0, "x": 100, "y": 100},
            {"type": "pointerDown", "button": 0},
            {"type": "pause", "duration": 500},
            {"type": "pointerMove", "duration": 1000, "origin": "pointer", "x": -50, "y": 0},
            {"type": "pointerUp", "button": 0}
        ]
    }, {
        "type": "pointer",
        "id": "finger2",
        "parameters": {"pointerType": "touch"},
        "actions": [
            {"type": "pointerMove", "duration": 0, "x": 100, "y": 100},
            {"type": "pointerDown", "button": 0},
            {"type": "pause", "duration": 500},
            {"type": "pointerMove", "duration": 1000, "origin": "pointer", "x": 50, "y": 0},
            {"type": "pointerUp", "button": 0}
        ]
    }]);
```

But this is for v4

**Comment by erwinheitzman** [comment]
Do you mean you are using v4 of webdriverio? If so, this version is many years old and is not supported for many years as well.

**Comment by dvngntfs** [comment]
I'm using the latest version, but i don't see any current documentation on
how to multitouch zoom on wdio's docs

**Reply by erwinheitzman** [reply]
I think https://webdriver.io/docs/api/browser/touchAction covers the example quite well

**Comment by dvngntfs** [comment]
Thanks.

i did see that and i was able to use the last example to drag and drop, but
this only covers one finger.

await browser.touchAction([
{ action: 'press', x: 200, y: 200 },
{ action: 'moveTo', x: 200, y: 300 },
'release'
])

i was hoping for documentation relating to multitouch pinch with two fingers

**Reply by erwinheitzman** [reply]
That's because I think you need this one for multi finger gestures: https://webdriver.io/docs/api/appium/#multitouchperform

**Reply by erwinheitzman** [reply]
Apologies that I did not share this the first time but my mobile knowledge is very limited
